openapi-diff/src/core/OpenApiDiff/OpenApiDiff.csproj

```diff
@@ -2,13 +2,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), common/common.proj ))/common/common.proj" />
   <Import Project="$(common)common-package-references.proj" />
   <Import Project="$(common)package-info.proj" />
-  <Import Project="$(common)package-fallback.proj" />
- 
+
   <PropertyGroup>
-    <RuntimeIdentifiers>win7-x64</RuntimeIdentifiers> 
+    <RuntimeIdentifiers>win7-x64</RuntimeIdentifiers>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
try to move this also to `netstandard2.0` if we can

Reply:
For `exe` type we cannot use `netstandard2.0` ( intended only for libraries). Since we are using this dill like an executable we need to stay to `netcoreapp2.0`